tests/geospatial/workloads/climatology.py

```diff
@@ -72,6 +72,7 @@ def rechunk_map_blocks(
     # Load dataset
     ds = xr.open_zarr(
         "gs://weatherbench2/datasets/era5/1959-2023_01_10-wb13-6h-1440x721.zarr",
+        chunks={"time": "auto"},
```

Comment:
This the a case where I think the naive user-code is what we want. This may be _slightly_ better but we want to optimize what most users will have in practice